### PR TITLE
Select id and randomnumber to match rules requirements

### LINKED ISSUE #6071: single db query of cpollcppsp
I was trying to figure out how other fw get more speed at single query tests,
and looking at https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/C%2B%2B/cpoll_cppsp/www/db_pg_async#L30
it doesn't retrieve the id column, according to https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#single-database-query the first item says that it's not acceptable to get only the random number column, did I read it right?

### PR DESCRIPTION
**Pull request by nbrady-techempower**
Resolves #6071 

I may have missed a few others, so feel free to add to this or comment below.

### CODE REVIEW

**Comment by dantti** [comment]
be careful, if you simply add id, in front of randomNumber chances are you are going to break the test.
Better after due column indexing.

**Comment by nbrady-techempower** [comment]
@dantti Good point. Looks like one test broke because of that

**Comment by joanhey** [comment]
More:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/3934542bd2aab91571c5590593fedac4aba275b0/frameworks/Python/apidaora/app.py#L15
https://github.com/TechEmpower/FrameworkBenchmarks/blob/34051fa94c83ce0914036bdd782abc0adc6e0a35/frameworks/Python/apidaora/coreapp.py#L20
https://github.com/TechEmpower/FrameworkBenchmarks/blob/811d7b821a0402d250ca8729164a6e4f16cc270a/frameworks/Python/sanic/app.py#L18
https://github.com/TechEmpower/FrameworkBenchmarks/blob/6ca1dd63dc3bb9dcf6d8421d45f3f5316a7dbe6f/frameworks/Python/blacksheep/app.py#L69
https://github.com/TechEmpower/FrameworkBenchmarks/blob/fef2b7a5265382f5e822edbc10581b2c764dec89/frameworks/Python/starlette/app.py#L13
https://github.com/TechEmpower/FrameworkBenchmarks/blob/87e6c1500f4c53d1754320e20ffc51567fd264e1/frameworks/Python/fastapi/app.py#L12
https://github.com/TechEmpower/FrameworkBenchmarks/blob/a160c71eb696798f128f4a91df03f49db1f59ace/frameworks/Python/responder/app.py#L10
https://github.com/TechEmpower/FrameworkBenchmarks/blob/f233196939c48d4a2192cc7c40be3b1b34a5600f/frameworks/Python/tornado/server_py3_uvloop.py#L18
https://github.com/TechEmpower/FrameworkBenchmarks/blob/fef2b7a5265382f5e822edbc10581b2c764dec89/frameworks/Python/uvicorn/app.py#L22
https://github.com/TechEmpower/FrameworkBenchmarks/blob/36e1e642d869f3e34ebad2816de9124f7299334a/frameworks/C/lwan/src/techempower.c#L55-L57
https://github.com/TechEmpower/FrameworkBenchmarks/blob/017292abbbe1c1233b8a6d177604beb8e613fb5d/frameworks/C%2B%2B/ulib/src/world.h#L337
https://github.com/TechEmpower/FrameworkBenchmarks/blob/b3a39dcd95b207cd2509d7bbf873a0dfb91097f5/frameworks/C%2B%2B/cpoll_cppsp/www/db_pg_async#L30

File: frameworks/Perl/mojolicious/app.pl [hunk 1]
```diff
@@ -80,23 +80,23 @@
   foreach (1 .. $q) {
     my $id = 1 + int rand 10_000;
 
-    push @queries, $self->helpers->pg->db->query_p('SELECT randomnumber FROM World WHERE id=?', $id)
+    push @queries, $self->helpers->pg->db->query_p('SELECT id,randomnumber FROM World WHERE id=?', $id)
```
Comment (joanhey) [inline frameworks/Perl/mojolicious/app.pl:83]: SELECT randomnumber, id FROM World WHERE id=?

I think that so will work.

**Review by joanhey — CHANGES_REQUESTED** [review]

**Comment by joanhey** [comment]
### Already failing before
D/Hunt  had problems starting or stopping (Add "tags": ["broken"] as is failing for the last 10 runs)
Perl/Mojolicious failed: query update
[quart] failed: fortune update
[quart-uvicorn] failed: update

https://tfb-status.techempower.com/results/9716e3cd-9e53-433c-b6c5-d2c48c9593c1

### Need changes from the developer
C++/Ulib, C++/cpoll_cppsp

Problem with Segmentation fault, that need to fix the _memset_

Still failing before:
[ulib-mysql] failed: fortune
[ulib-postgres] failed: fortune
[ulib-postgres_fit] failed: fortune
[cpoll_cppsp-raw] failed: fortune

**Comment by nbrady-techempower** [comment]
Going to get these in for now and fix the notes and more in another PR

**Pull request merged**